server/src/main/java/org/spine3/server/entity/EntityRepository.java

```diff
@@ -91,22 +108,168 @@ public E load(I id) {
         return entity;
     }
 
+    /**
+     * Finds the entity with the passed ID.
+     *
+     * <p>As opposed to {@link #load(Object)}, the invocation of this method never leads to creation of a new entity.
+     *
+     * <p>NOTE: The storage must be assigned before calling this method.
+     *
+     * @param id the id of the entity to find.
+     * @return the entity or {@code null} if there's no entity with such id
+     */
+    @CheckReturnValue
+    @Nullable
+    public E find(I id) {
+        return this.load(id);
+    }
+
+
+    /**
+     * Finds all the entities in this repository with IDs, contained within the passed {@code Iterable}.
+     *
+     * <p>Same as calling <pre>{@code findBulk(id, FieldMask.getDefaultInstance());}</pre>
+     *
+     * @param ids entity IDs to search for
+     * @return all the entities in this repository with the IDs matching the given {@code Iterable}
+     * @see #findBulk(Iterable, FieldMask)
+     */
+    @CheckReturnValue
+    public ImmutableCollection<E> findBulk(Iterable<I> ids) {
```

Comment:
Because `findAll` takes `EntityFilters` as a parameter.
And `findBulk` takes ID collection.

In fact, right now `findAll(EntityFilters)` is a convenience wrapper over `findBulk(ids)` for those, who deals with the `Query` instance — hence, has the `EntityFilters` object, not IDs straightaway.